python/paddle/fluid/layers/tensor.py

```diff
@@ -214,47 +256,48 @@ def concat(input, axis=0, name=None):
 
 def tensor_array_to_tensor(input, axis=1, name=None):
     """
-    This function concatenates the input LodTensorArray along the axis mentioned
-    and returns that as the output.
-
-    A simple example as below:
-
-    .. code-block:: text
-
-        Given:
-
-        input.data = {[[0.6, 0.1, 0.3],
-                       [0.5, 0.3, 0.2]],
-                      [[1.3],
-                       [1.8]],
-                      [[2.3, 2.1],
-                       [2.5, 2.4]]}
-
-        axis = 1
-
-        Then:
-
-        output.data = [[0.6, 0.1, 0.3, 1.3, 2.3, 2.1],
-                       [0.5, 0.3, 0.2, 1.8, 2.5, 2.4]]
-
-        output_index.data = [3, 1, 2]
+    This OP concatenates the input LodTensorArray along the axis.
 
     Args:
-        input(list): Input LodTensorArray
-        axis(int): Integer axis along which the tensors will be concatenated
-        name(str|None): A name for this layer(optional). If set None, the layer
-                       will be named automatically.
+        input(list): A LodTensorArray with data type float32, float64, int32,
+            int64.
+        axis(int, optional): Axis to compute indices along. The effective range
+            is [-R, R), where R is Rank(x). when axis<0, it works the same way
+            as axis+R. Default is 1.
+        name (str, optional): The default value is None. Normally there is no
+            need for user to set this property. For more information, please
+            refer to :ref:`api_guide_Name`.
 
     Returns:
-        Variable: Output variable of the concatenation
-        Variable: The input LodTensorArray items' dims along the axis
+        Variable: A LoDTensor with the same data type with input
```

Comment:
A LoDTensor  which has the same data type as input's

Reply:
Done